lib/assembly-image/image.rb

```diff
@@ -94,16 +94,6 @@ def jp2_filename
       File.extname(path).empty? ? "#{path}.jp2" : path.gsub(File.extname(path), '.jp2')
     end
 
-    # Returns the full DPG equivalent jp2 path and filename that would match with the given image
-    #
-    # @return [string] full DPG equivalent jp2 path and filename
-    # Example:
-    #   source_img=Assembly::Image.new('/input/path_to_file.tif')
-    #   puts source_img.jp2_filename # gives /input/path_to_file.jp2
-    def dpg_jp2_filename
-      jp2_filename.gsub('_00_', '_05_')
-    end
-
     # Create a JP2 file for the current image.
```

Comment:
@calavano this was removed in your test PR as well, I assume it's no longer needed?

Reply:
DPG hasn't used this naming convention in a long time, it shouldn't be needed. I believe that there will be a couple other sections of code that should be removed that call out using the old DPG file naming (or might be in assembly-objectfile).